src/include/parser/expression_defs.h

```diff
@@ -3,9 +3,13 @@
 #include <string>
 
 #include "common/enum_defs.h"
+#include "common/strong_typedef.h"
 
 namespace noisepage::parser {
 
+// It's important that alias_oid_t is the same size as pointers because we often convert a pointer to an alias_oid_t
+STRONG_TYPEDEF_HEADER(alias_oid_t, size_t);
```

Comment:
namespace `std` re-exports the C library types like `size_t`, and real C++ devs always use `std` where possible.

Reply:
Can you dumb this down for me, you're saying I should use `std::size_t` instead of `size_t`?

Reply:
I think that's what you're saying, I made the change.